toolchain/check/import_ref.cpp

```diff
@@ -32,16 +32,26 @@ static auto InternalAddImportIR(Context& context, SemIR::ImportIR import_ir)
   return context.import_irs().Add(import_ir);
 }
 
-auto SetApiImportIR(Context& context, SemIR::ImportIR import_ir) -> void {
+// Sets the IR for a specific ImportIRId. Should be called in the right order in
+// order to ensure the correct ID is assigned.
```

Comment:
Maybe clarify where to find the right order?
```suggestion
// Sets the IR for a specific ImportIRId. Ordering of calls is significant; see `SetApiImportIRs`.
```

Or, perhaps:

```suggestion
// Adds a special-cased IR and verifies it received the correct ID.
```

Reply:
Done (2nd suggestion).